tasks/package.yml

```diff
@@ -5,12 +5,28 @@
     pkg: python-pycurl
     state: present
 
-- name: nodejs | Add the nodejs repository
-  apt_repository:
-    repo: ppa:chris-lea/node.js
+- name: node.js | get release
```

Comment:
I think this is available in local facts, under `ansible_distribution_release` see http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts